tests/foreman/cli/test_computeresource.py

```diff
@@ -84,23 +78,16 @@ def test_list(self):
         @Assert: Compute resource List is displayed
 
         """
-        result_create = make_compute_resource({
+        comp_res = make_compute_resource({
             'provider': FOREMAN_PROVIDERS['libvirt'],
-            'url': "qemu+tcp://%s:16509/system" %
+            'url': 'qemu+tcp://%s:16509/system' %
             conf.properties['main.server.hostname']})
-        self.assertTrue(result_create['name'],
-                        "ComputeResource create - has name")
-        result_list = ComputeResource.list({'search': "name=%s" %
-                                            result_create['name']})
-        self.assertEquals(result_list.return_code, 0,
-                          "ComputeResource list - exit code")
-        self.assertTrue(len(result_list.stdout) > 0,
-                        "ComputeResource list - stdout has results")
-        stdout = ComputeResource.exists(
-            search=('name', result_create['name'])).stdout
-        self.assertTrue(
-            stdout,
-            "ComputeResource list - exists name")
+        self.assertTrue(comp_res['name'])
```

Comment:
Did you really mean to assert this as a `bool` or was the intention to make sure that `name` matches the string `"ComputeResource create - has name"`?